io.github.fralonra.WgShadertoy.yml

```diff
@@ -0,0 +1,35 @@
+app-id: io.github.fralonra.WgShadertoy
+runtime: org.freedesktop.Sdk
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: wgshadertoy
+
+finish-args:
+  - --device=dri
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission

Reply:
can't this be narrowed down? When this happen is irrelevant.

Reply:
The app is about to create and edit [`wgs`](https://github.com/fralonra/wgshadertoy#wgs-format) file and might needs to upload images.
What is the recommanded filesystem permission?

Reply:
@hfiguiere Hi, I just removed all `--filesystem` permissions for my local build, but the app worked fine as it could access the whole file system. Is that worked as intended?